scripts/split_dataset.py

```diff
@@ -126,8 +126,13 @@ def main():
                 assert trainset_size + validset_size + testset_size == nb_examples
 
                 trainset_indices = indices[:trainset_size]
-                validset_indices = indices[trainset_size:-testset_size]
-                testset_indices = indices[-testset_size:]
```

Comment:
Good catch. If you want a one-liner you could do
`indices[trainset_size:nb_examples-testset_size]`a and similar for the other one.